src/modules/form/components/EditRecord.tsx

```diff
@@ -143,15 +140,13 @@ const EditRecord = <RecordType extends SubmitFormAllowedTypes>({
         >
           {title}
         </Box>
-        <Grid container spacing={2} sx={{ pb: 20, mt: 0 }}>
-          <FormNavigation
-            items={leftNavItems}
-            top={top}
-            {...FormNavigationProps}
-          >
-            {form}
-          </FormNavigation>
-        </Grid>
+        <FormNavigationContainer
+          navItems={leftNavItems}
+          navScrollOffset={top}
+          {...FormNavigationProps}
```

Comment:
One visual result of this refactor is moving the `EditRecord` after-nav action buttons into the FormStepper's Paper, like this:

Before:
![Screenshot 2025-01-08 at 11 38 16 AM](https://github.com/user-attachments/assets/96850d48-5636-49b2-ad4b-278ff7451ebf)

After:
![Screenshot 2025-01-08 at 11 38 22 AM](https://github.com/user-attachments/assets/5730ebfa-d304-49d6-acc4-84dda6717596)

I think it's an OK change, even good since it's now consistent with Assessments. But let me know if you would prefer to keep separate or loop in design!

Reply:
looks fine to me, since its consistent with assessments I think it makes sense